backend/tests/v5/unit/models/fileRefs.test.js

```diff
@@ -154,10 +154,8 @@ const testFetchFileStream = () => {
 
 		test('should download the revision files using grid fs if entry has invalid type', async () => {
```

Comment:
since we expect it to throw error, is the description valid ?

Reply:
Yup, the behaviour changed since we no longer store a backup copy